gatsby-node.js

```diff
@@ -1,27 +1,15 @@
 const {
-  getNodeSlug,
   setPageContext,
   removePageTrailingSlash
 } = require('./src/gatsby/common')
-const { createPages: createDocPages } = require('./src/gatsby/doc')
-const { createPages: createBlogPages } = require('./src/gatsby/blog')
 
-exports.onCreateNode = ({ node, actions, getNode }) => {
-  const { createNodeField } = actions
+const buildModelApi = require('./src/gatsby/utils/models')
```

Comment:
see my comment below in the `buildModelApi` definition, I think we can simplify and not create multi-layered builder to this simple task